packages/react/src/components/ModalWrapper/ModalWrapper-story.js

```diff
@@ -52,8 +52,8 @@ const props = () => {
       'Modal contains scrollable content (hasScrollingContent)',
       true
     ),
-    renderTriggerButtonIcon: typeof iconToUse === 'function' && iconToUse,
-    triggerButtonIcon: typeof iconToUse !== 'function' && iconToUse,
+    renderTriggerButtonIcon:
+      typeof iconToUse === 'function' ? iconToUse : undefined,
```

Comment:
Quick note: the line break + indentation is prettier formatting from the pre-commit hook 😅